data/json/vehicles/military.json

```diff
@@ -1,3 +1,8 @@
+Cataclysm: Dark Days Ahead JSON Web Linting Tool
+This is a tool to help modders and editors of the open source game Cataclysm: Dark Days Ahead write JSON in the game's expected format.
+
+Paste some JSON into the field below and click "Lint" to run an autoformatter against it.
+
```

Comment:
you copypasted too much :D

Reply:
Yea, i just seen that! I thought if i used the web-tools i would have less problems.. Apparently not! xD
I was going to ask you above but forgot, do you have any suggestions to where to put it?I am thinking to add it on to the `hazardous_waste_sarcophagus.json` and `mass_grave.json` and have placed it on every array in `vehicle_groups.json` which contains the original. What do you think?

Reply:
Yea, i just seen that! I thought if i used the web-tools i would have less problems.. Apparently not! xD

I was going to ask you above but forgot, do you have any suggestions to where to put it?  Originally i'm just using it to be half backed in to a warehouse at an Old Guard base as if it's being unloaded, pure decoration, and not to spawn anywhere. I am thinking to add it on to the `hazardous_waste_sarcophagus.json` and `mass_grave.json`,  what do you think?
What I have done is placed it on every array in `vehicle_groups.json` which contains the original.

Reply:
search all places where `military_cargo_truck` exist, and if it's a vehicle group, add it here

Reply:
> search all places where `military_cargo_truck` exist, and if it's a vehicle group, add it here

More or less what I have done, thanks, except I don't want to put it in the roadblock because i'm not sure it fits there nor the military base map because i'm not very familiar with it (and the map is something i am trying to highly avoiding getting into more generally so the project of an Old Guard nucleus doesn't widen and widen!).

Reply:
don't touch maps, the way they spawn vehicles is super dumb and obscure

Reply:
It's not that difficult i find it just tedious personally, the main issue is stuff like collisions with terrains and furnitures. 
What I think I am going to do is create a vehicle-group containing the variations of truck and replace the original truck on the maps with the aforementioned group.  :)
Anyway sorry for the delay I realized i should probably download the latest experimental before doing stuff with vehicle groups.

Reply:
That is a good thing to do, but remember that vehicle spawn rely on the vehicle coordinate 0:0, which is not standardized, and can be anywhere in the vehicle, and the game will spawn car in a wall without any safe checks